static/styles/rendered_markdown.scss

```diff
@@ -470,6 +470,10 @@
     }
 }
 
+.preview_content .copy_codeblock {
```

Comment:
Let's copy your commit message to a comment here; we should be in the habit of commenting details like this.